Source/MonoGame.Extended/MonoGame.Extended.csproj

```diff
@@ -17,8 +17,18 @@
     <PackageIconUrl>https://raw.githubusercontent.com/craftworkgames/MonoGame.Extended/master/Logos/logo-nuget-128.png</PackageIconUrl>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Remove="Collections\BitVector32.cs" />
+  </ItemGroup>
+
   <ItemGroup>
     <PackageReference Include="MonoGame.Framework.Portable" Version="3.6.0.1625" />
     <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
   </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="System.Numerics.Vectors">
+      <HintPath>..\..\..\..\..\..\..\Program Files\dotnet\sdk\NuGetFallbackFolder\microsoft.netcore.app\2.0.0\ref\netcoreapp2.0\System.Numerics.Vectors.dll</HintPath>
```

Comment:
This hint path is interesting? It references a DLL on the machine rather than a package on NuGet?

Reply:
I let VS add the reference as I didn't know where it was... Apparently I'll have to look at that haha

Reply:
So, now that I'm looking at this, this is just to reference the new .NET `Matrix3x2` and `Vector2` which we aren't using. This is COMPLETELY unneeded... *facepalm*